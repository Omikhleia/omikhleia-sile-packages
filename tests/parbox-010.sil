\begin{document}
\neverindent
\nofolios
\script[src=packages/parbox]
\script[src=packages/lorem]
% Test case 010 = From the package documentation, alignment without struts

(1A) \parbox[width=37%fw, valign=top, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\smallskip

(1B) \parbox[width=37%fw, valign=bottom, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\smallskip

(1C) \parbox[width=37%fw, valign=middle, border=0.5pt]{one\par
\lorem[words=8]\par
two}

\end{document}