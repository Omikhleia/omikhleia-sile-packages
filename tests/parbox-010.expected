Set paper size 	595.275597	841.8897729
Begin page
Mx 	29.7638
My 	49.9070
Set font 	Gentium Plus;10;400;;normal;;LTR
T	11 w=3.1689	(()
Mx 	32.9327
T	20 36 w=10.6445	(1A)
Mx 	43.5773
T	12 w=3.1689	())
Draw line	49.4238	49.9070	198.2268	0.5000
Draw line	49.4238	90.7410	198.2268	0.5000
Draw line	49.4238	49.9070	0.5000	41.3340
Draw line	247.6506	49.9070	0.5000	41.3340
Mx 	49.4238
My 	54.5945
T	82 81 72 w=15.1660	(one)
My 	66.5945
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	78.2510
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.9278
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	134.3331
T	86 76 87 w=10.0146	(sit)
Mx 	148.8243
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.9845
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.6213
T	86 68 w=8.4521	(sa)
Mx 	231.0734
T	71 76 83 w=13.2080	(dip)
Mx 	244.2814
T	16 w=3.3691	(-)
Mx 	49.4238
My 	78.5945
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	73.4215
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	49.4238
My 	90.5945
T	87 90 82 w=15.5322	(two)
Mx 	29.7638
My 	135.5750
T	11 w=3.1689	(()
Mx 	32.9327
T	20 37 w=10.1904	(1B)
Mx 	43.1232
T	12 w=3.1689	())
Draw line	48.9697	94.7410	198.2268	0.5000
Draw line	48.9697	135.5750	198.2268	0.5000
Draw line	48.9697	94.7410	0.5000	41.3340
Draw line	247.1965	94.7410	0.5000	41.3340
Mx 	48.9697
My 	99.4285
T	82 81 72 w=15.1660	(one)
My 	111.4285
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	77.7969
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.4737
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	133.8790
T	86 76 87 w=10.0146	(sit)
Mx 	148.3703
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.5305
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.1672
T	86 68 w=8.4521	(sa)
Mx 	230.6194
T	71 76 83 w=13.2080	(dip)
Mx 	243.8274
T	16 w=3.3691	(-)
Mx 	48.9697
My 	123.4285
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	72.9675
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	48.9697
My 	135.4285
T	87 90 82 w=15.5322	(two)
Mx 	29.7638
My 	161.5300
T	11 w=3.1689	(()
Mx 	32.9327
T	20 38 w=10.0537	(1C)
Mx 	42.9864
T	12 w=3.1689	())
Draw line	48.8330	141.1130	198.2268	0.5000
Draw line	48.8330	181.9470	198.2268	0.5000
Draw line	48.8330	141.1130	0.5000	41.3340
Draw line	247.0598	141.1130	0.5000	41.3340
Mx 	48.8330
My 	145.8005
T	82 81 72 w=15.1660	(one)
My 	157.8005
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	77.6602
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	107.3370
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	133.7423
T	86 76 87 w=10.0146	(sit)
Mx 	148.2336
T	68 80 72 87 w=20.6836	(amet)
Mx 	173.3937
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	222.0305
T	86 68 w=8.4521	(sa)
Mx 	230.4826
T	71 76 83 w=13.2080	(dip)
Mx 	243.6907
T	16 w=3.3691	(-)
Mx 	48.8330
My 	169.8005
T	86 70 76 81 74 w=21.3379	(scing)
Mx 	72.8308
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	48.8330
My 	181.8005
T	87 90 82 w=15.5322	(two)
End page
Finish
